src/lib/tabs/tab-group.ts

```diff
@@ -157,19 +155,22 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
    * a new selected tab should transition in (from the left or right).
    */
   ngAfterContentChecked(): void {
-    // Clamp the next selected index to the bounds of 0 and the tabs length. Note the `|| 0`, which
-    // ensures that values like NaN can't get through and which would otherwise throw the
-    // component into an infinite loop (since Math.max(NaN, 0) === NaN).
+    // Clamp the next selected index to the boundsof 0 and the tabs length.
```

Comment:
Did the comment change?

Reply:
No, I *had* changed it and had to reformat it to fix length but then backed that out and didn't undo the reformat. Sorry.